sources/datasource/kubelet.go

```diff
@@ -90,7 +90,7 @@ func (self *kubeletSource) getContainer(url string, start, end time.Time, resolu
 }
 
 func (self *kubeletSource) GetContainer(host Host, start, end time.Time, resolution time.Duration) (container *api.Container, err error) {
-	url := fmt.Sprintf("http://%s:%s/%s", host.IP, host.Port, host.Resource)
+	url := fmt.Sprintf("https://%s:%s/%s", host.IP, host.Port, host.Resource)
```

Comment:
I don't think this will just work. You need to configure an http.Client that will connect to the kubelet without verifying the server's SSL cert (by setting InsecureSkipVerify to true in the TLSClientConfig).